fossunited/fossunited/doctype/foss_user_profile/foss_user_profile.json

```diff
@@ -60,7 +60,8 @@
    "fieldname": "email",
    "fieldtype": "Data",
    "label": "Email",
-   "options": "Email"
+   "options": "Email",
+   "unique": 1
```

Comment:
The fields `email` and `full_name` are being fetched from the "User" core doctype of Frappe Framework. So it is not at all needed for it to be unique here.

The idea is when a person signs up for the platform, a core "User" doctype will be created and a seperate FOSS User Profile will be created related to this core doctype. While "User" doctype can handle all the permissions/roles/password related stuff of user, FOSS User Profile will be used to carry the additional info about the user (such as Experience etc.)

Reply:
@Mangeshrex I have conveyed the same him. There are a few redundant fields, eg: the reviewers doctype.
UserProfile should ideally be the only single doctype where all suerdata and metadata must be fetched

Reply:
I did not see any validation, Doc Event to auto fetch the same. Maybe I am lacking context regarding your control flow.
Will wait for a bit :)

Reply:
Most fields that may seem redundant are there only because the data is being fetched into them from the User Doctype.  Maybe I can make then read-only so that people do not get confused in the future. 

The reason you can't see the validations is probably becauseit is being handled through Frappe Desk. For Ex: The email field in the FOSS User Profile has this:
![image](https://github.com/fossunited/fossunited/assets/73123690/24219760-4264-4f76-9a0f-920bf921253a)
 
This let's the field to autofetch the email value from the User link field every time a doctype element is created/saved.